docs/DeveloperGuide.md

```diff
@@ -930,36 +930,100 @@ testers are expected to do more *exploratory* testing.
 ### Sorting clients
 
 ### Updating last met
-Command Format: `met index d/YYYY-MM-DD`<br>
-Assumptions: Today is 13 April 2024, date chosen must not be in the future.<br>
-Desired Outcome: Updating the last met date of the 3rd client to 11 April 2024.<br>
-1. Correct Test Case: `met 3 d/2024-04-11`
-2. Invalid Test Case: `met 3 d/11-04-2024` wrong date format
 
+1. Updating the last met of a client while all clients are being shown
+   1. Prerequisites: List all clients using the `list` command. Multiple clients in the client list. Ensure the date chosen is not in the future.
+   For the test cases, we assume that today is 13 April 2024.
+
+   2. Test case: `met 1 d/2024-04-11`<br>
+      Expected: Last met successfully updated for first client. Success message shown in the status message.
+   
+   3. Test case (Invalid Date Format): `met 3 d/11-04-2024`<br>
+      Expected: Last met not updated for any client. Error details in status message.
+   
+   4. Test case (Missing Parameters): `met d/2024-04-11`, `met 3` or any command with missing parameters<br>
+      Expected: Similar to previous
+   
+   5. Test case (Repeated Parameters): `met 3 d/2024-04-11 d/2024-04-11` or any command with repeated parameter<br>
+      Expected: Similar to previous.
+   
+   6. Test case (Invalid Index): `met x d/2024-04-11` (where x is smaller or larger than the list size)<br>
+      Expected: Similar to previous
+   
+   7. Test Case (Future Date, Invalid Date): `met 3 d/2024-04-20`
```

Comment:
Should we be setting a much later date like in 2025? The bug testers might still be testing it after 20 April.